### PR TITLE
add missing for loop

### PR DESCRIPTION
**Pull request by Marlene-M-Hirose**
Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-1194)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/adjust_derived/adjust_deliverables_v1/query.py [hunk 1]
```diff
@@ -264,11 +264,12 @@ def main():
...
             # This section writes the tmp json data into a temp CSV file which will then be put into a BigQuery table
             adjust_data = clean_json(query_export)
-            data.append(adjust_data)
+            for datum in adjust_data:
+                data.append(datum)
         else:
             print(f'no data for {app["app_name"]} today')
 
-    upload_to_bigquery(data, args.project, args.dataset, args.date)
+    # upload_to_bigquery(data, args.project, args.dataset, args.date)
```
Comment (ANich) [inline sql/moz-fx-data-shared-prod/adjust_derived/adjust_deliverables_v1/query.py:272]: Is this supposed to be commented?

File: sql/moz-fx-data-shared-prod/adjust_derived/adjust_deliverables_v1/query.py [hunk 2]
```diff
@@ -264,11 +264,12 @@ def main():
         if query_export is not None:
             # This section writes the tmp json data into a temp CSV file which will then be put into a BigQuery table
             adjust_data = clean_json(query_export)
-            data.append(adjust_data)
+            for datum in adjust_data:
+                data.append(datum)
```
Comment (ANich) [inline sql/moz-fx-data-shared-prod/adjust_derived/adjust_deliverables_v1/query.py:268]:
```suggestion
            data.extend(adjust_data)
```

**Review by ANich — COMMENTED** [review]
One suggestion and a question about the upload

File: sql/moz-fx-data-shared-prod/adjust_derived/adjust_deliverables_v1/query.py (hunk 1 above)
Comment (Marlene-M-Hirose) [reply sql/moz-fx-data-shared-prod/adjust_derived/adjust_deliverables_v1/query.py:272]: whoops, thanks!

**Review by ANich — APPROVED** [review]

**Comment by Marlene-M-Hirose** [comment]
Originally, the code had a for loop to traverse through a list to write data to a csv. Somehow the for loop disappeared in one of the updates and this caused the code to fail. This PR exists to add the for loop back in as well as rename folders to more closely match the table name

**Pull request merged**